### DISCUSSION TITLE
Tailwind 3 webpack bundles include too much chatter

### DISCUSSION THREAD

**Discussion by mitchellworks** [tailwindlabs/tailwindcss#7266] [Help] [answered]
**What version of Tailwind CSS are you using?**

3.0.16

**What build tool (or framework if it abstracts the build tool) are you using?**

"postcss": "8.4.5",
"webpack": "5.39.1"

**What version of Node.js are you using?**

14.15.4

**What browser are you using?**

N/A

**What operating system are you using?**

macOS

**Reproduction URL**

See screenshot below

**Describe your issue**

### Discussed in https://github.com/tailwindlabs/tailwindcss/discussions/7224

<div type='discussions-op-text'>

<sup>Originally posted by **mitchellworks** January 26, 2022</sup>
Hi all!
I have been upgrading from Tailwind 2 to Tailwind 3, which has gone smoothly. However, I noticed that all of my webpack bundle sizes went up.

When I look at the bundles, I see that around the area where Tailwind inserts a link to its MIT License, the new Tailwind 3 adds a bunch of extra verbiage, which I can only think is a mistake. 

Here are screenshots of the same bundle, from TW2 and then TW3.

![image](https://user-images.githubusercontent.com/16616554/151235769-3d2f67eb-5adc-4bdc-be8f-88a012af6499.png)

![image](https://user-images.githubusercontent.com/16616554/151235807-8e402833-8ffb-49eb-b85f-0f122b206d27.png)

</div>

**Comment by adamwathan** [comment, marked as answer]
Hey! Can you provide a reproduction? There should be no difference here, your minifier should still remove those comments if it was removing them before.

Our production CSS for tailwindcss.com doesn't include those comments for example because cssnano removes them:

https://tailwindcss.com/_next/static/css/eaed56ccdfc5b5f1.css

```
/*
! tailwindcss v3.0.17 | MIT License | https://tailwindcss.com
*/*,:after,:before{box-sizing:border-box;border:0 solid #e5e7eb}:after,:before{--tw-content:""}html{line-height:1.5;-webkit-text-size-adjust:100%;-o-tab-size:4;tab-size:4;font-family:Inter var,ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji}body{margin:0;line-height:inherit}hr{height:0;color:inherit;border-top-width:1px}abbr:where([title]){-webkit-text-decoration:underline dotted;text-decoration:underline dotted}h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight:inherit}a{color:inherit;text-decoration:inherit}b,strong{font-
...
```

**Reply by mitchellworks** [reply]
Thank you. Previously we didn't seem to need cssnano, but adding cssnano to postcss.config.js did remove the extra chatter for tw3. Thanks!
